dvc/repo/add.py

```diff
@@ -43,7 +43,14 @@ def check_recursive_and_fname(args):
 
 
 def transform_targets(args):
-    args.targets = ensure_list(args.targets)
+    from collections import Counter
+
+    counter = Counter(ensure_list(args.targets))
+    dupes = counter.most_common(n=2)
+    if dupes:
+        msg = ", ".join(f"[b]{key}[/]" for key, _ in dupes)
+        ui.error_write(f"ignoring duplicated targets: {msg}", styled=True)
```

Comment:
hmm, the capitalization of message (or, not) could be another topic for the UI discussion.